infra/helper.py

```diff
@@ -369,7 +369,16 @@ def get_parser():  # pylint: disable=too-many-statements,too-many-locals
   index_parser = subparsers.add_parser('index', help='Index project.')
   index_parser.add_argument(
       '--targets', help='Allowlist of targets to index (comma-separated).')
+  index_parser.add_argument('--shell',
+                            action='store_true',
+                            help='Run /bin/bash instead of the indexer.')
+  index_parser.add_argument('--docker_args',
```

Comment:
I would make that `nargs='*'` as well, that saves a lot of headaches with shell escaping and argument splitting.

The Docker arguments can as well be written with an equal sign, like `--volume=/hostdir:/containerdir`.

Reply:
P.S. See documentation at https://docs.python.org/3/library/argparse.html#nargs - `nargs` works for named flags as well, not just for positional arguments.

Reply:
Thanks for the suggestion, done!